clang/test/SemaCXX/cxx2a-no-unique-address.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %clang_cc1 -std=c++2a %s -verify -triple x86_64-linux-gnu
-// RUN: %clang_cc1 -std=c++2a %s -verify=unsupported -triple x86_64-windows
```

Comment:
This also appears to be a bug; given the Windows triple, this should still be giving "unknown attribute ignored" warnings.